.github/workflows/run-match.yml

```diff
@@ -0,0 +1,75 @@
+name: Simulator match
+
+on:
+  push: {}
+  release:
+    types: [ created ]
+
+jobs:
+  run-comp-match:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ${{ github.workspace }}
+    strategy:
+      matrix:
+        webots-version: ["2020b-rev1", "2021a"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          sudo apt-get update
+          sudo apt-get install --yes wget zip pulseaudio
+      - name: Download Webots
+        run: |
+          wget -O ./webots.deb \
+          https://github.com/cyberbotics/webots/releases/download/R${{ matrix.webots-version }}/webots_${{ matrix.webots-version }}_amd64.deb
+      - name: Install Webots & Xvfb
+        run: |
+          sudo apt-get install --yes xvfb ./webots.deb
```

Comment:
Probably worth just installing `xvfb` with the installs on line 28? Keep things separate?

Reply:
Certainly, this was just a hold over from some caching attempts